events/tests/test_event_auth_data.py

```diff
@@ -1,48 +0,0 @@
-# -*- coding: utf-8 -*-
-import pytest
-from .utils import versioned_reverse as reverse
-
-from .test_event_get import get_detail
-
-
-@pytest.mark.django_db
-def test__user_sees_auth_fields(api_client, event, user, user2, organization, organization2):
```

Comment:
I guess these two asserts (`assert response.data.get('publication_status') == "public"` and `assert response.data.get('publication_status') is None`) should be included somewhere in the new tests, because they test field visibility for admins and random users?

Reply:
Oh, there they are, the diff was so large it wasn't loaded! My bad :) So how about removing `test__a_non_admin_cannot_update_an_event` in `test_event_put.py` still?

Reply:
True that, I'll remove it.